surefire-integration-tests/src/test/java/org/apache/maven/surefire/its/fixture/HelperAssertions.java

```diff
@@ -159,4 +164,10 @@ public static IntegrationTestSuiteResults parseReportList( List<ReportTestSuite>
             throw new RuntimeException( "Couldn't parse XML reports", e );
         }
     }
+
+    public static void assumeJavaVersion(JavaVersion version)
+    {
+        assumeThat( "java.specification.version: ",
+                SystemUtils.JAVA_SPECIFICATION_VERSION, is( greaterThanOrEqualTo( version.toString() ) ) );
```

Comment:
Unfortunately `org.apache.commons.lang3.JavaVersion` is not `Comparable`. How would this work with string `1.10`? JDK 9, I guess, returns `9` instead of `1.9`. Did you try? If so then such lexical comparison should be fine. JDK 10 I hope will follow the same pattern.  Version `10.1` bigger than `10`. Are you facing any problem with lexicographic comparison?